### PR TITLE
DDT-942: IT for booking summaries endpoint

### PR DESCRIPTION
**Pull request by Abdiiir**

### CODE REVIEW

File: src/test/java/bkg/b2/bookingSummariesIT.java [hunk 1]
```diff
@@ -0,0 +1,91 @@
+package bkg.b2;
```
Comment (mp-assco) [inline src/test/java/bkg/b2/bookingSummariesIT.java:1]: I assume this is supposed to be v2?

Comment (Abdiiir) [reply src/test/java/bkg/b2/bookingSummariesIT.java:1]: i went with b2 due to the beta naming, but yeah i guess V2 makes more sense

Comment (mp-assco) [reply src/test/java/bkg/b2/bookingSummariesIT.java:1]: I'm unsure if it's v/b1 or 2. Swagger is v1 for BKG and v2 for eBL.

Comment (pimg) [reply src/test/java/bkg/b2/bookingSummariesIT.java:1]: no BKG is version 1 so it should go into v1

Comment (mp-assco) [reply src/test/java/bkg/b2/bookingSummariesIT.java]: Done

**Review by pimg — APPROVED** [review]

**Pull request merged**